index.js

```diff
@@ -67,10 +67,16 @@ async function main(outputDirectory) {
 				outputDirectory : path.join(outputDirectory, tag || `untagged`)
 
 			if (trashed) {
-				return fs.unlink(path.join(destinationDirectory, filename))
+				const filePath = path.join(destinationDirectory, filename)
+
+				fs.stat(filePath).then(() => {
```

Comment:
The `writeFile` down below will still get called if `trashed` is true in this case.

I think this whole arrow function should just be changed to `async` and then this block can be something like

```js
if (trashed && await fs.stat) {
	return fs.unlink
} else if (!trashed) {
	return fs.writeFile
}
```